docs/products/opensearch/concepts/indices.md

```diff
@@ -2,199 +2,169 @@
+used to calculate recommended plan sizes.
 
 :::warning
-The Dashboards from Aiven for OpenSearch are not compatible across minor
-versions. Because of this, if existing current nodes of your service
-instance are of an older version, you can expect a downtime during
-migration or plan changes.
+Dashboards from Aiven for OpenSearch are not compatible across minor versions of
+OpenSearch. If your service instance runs an older OpenSearch version, expect downtime
+during migration or plan changes.
 :::
 
-## Using patterns to set index retention
+## Set index retention with patterns
+
+The standard approach to index retention is setting a maximum number of indices to store.
+When that number is exceeded, the oldest index is deleted. However, this approach has
+limitations. For example, you may need to set different maximums for different types of
+indices, such as 5 for one type and 8 for another.
+
+### Create glob-style patterns
+
+Aiven for OpenSearch allows you to create
+[glob-style](https://en.wikipedia.org/wiki/Glob_(programming)) patterns to set unique
+maximums for each pattern. For example:
+
+- `logs`: matches `logs` but not `logsfoo` or `foologs`
+- `logs*`: matches `logsfoo` and `logs_foo_bar` but not `foologs`
+- `*_logs_*`: matches `foo_logs_bar` but not `foologsbar`
+- `logs.?`: matches `logs.1` but not `logs.11`
+
+### Use the catch-all pattern
+
+You can also use `*` as a catch-all pattern to match all indices. This pattern iterates
+through all your patterns, so consider the impact carefully before using it.
+
+### Example scenario
 
-The standard index retention approach is setting the maximum number of
-indices to store. On exceeding that number, the oldest index is deleted.
-However, this has some limitations. For example, you might want to set
-the maximum to 5 for one type of index and 8 for another type.
+Consider a scenario where you have the following indices:
+- Log indices: `logs.1`, `logs.2`, up to `logs.34`
+- Test indices: `test.1`, `test.2`, `test.3`, intended for testing but not in active use
 
-Aiven for OpenSearch services gives you the option to create
-[glob-style](https://en.wikipedia.org/wiki/Glob_(programming)) patterns
-and set a unique maximum for each pattern.
+If you create:
 
-:::note[Pattern examples]
--   `logs`: matches `logs` but not `logsfoo` or `foologs`
--   `logs*`: matches `logsfoo` and `logs_foo_bar` but not `foologs`
--   `*_logs_*`: matches `foo_logs_bar` but not `foologsbar`
--   `logs.?`: matches `logs.1` but not `logs.11`
+- A `logs.*` pattern with a maximum of 8 indices
+- A `*` pattern with a maximum of 3 indices
+
+Aiven for OpenSearch processes these patterns in sequence:
+
+1. It first retains the 8 newest log indices according to the `logs.*` pattern.
+1. Next, it applies the `*` pattern, which affects all indices. Since the test indices
+   are the newest, they are retained, and the log indices are deleted.
```

Comment:
Let's never write what's first and next with words. That's already contained in the ordered list numbering ✌️

Reply:
done